.npmignore

```diff
@@ -0,0 +1 @@
+src/
```

Comment:
It’s better to use `files` field in package.json and explicitly specify files you want to publish.